services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2218,7 +2218,17 @@ private void saveFile(Node parent, UploadResource resource, String user, List<St
                                                           .append(PermissionType.ADD_NODE)
                                                           .append(",")
                                                           .append(PermissionType.SET_PROPERTY)
+                                                          .append(",")
+                                                          .append(PermissionType.REMOVE)
                                                           .toString();
+      try {
+        if (PermissionUtil.canChangePermission(fileNode)) {
+          setUserPermission(fileNode, user, perm.split(","));
+          fileNode.getSession().save();
```

Comment:
No need for this instruction, the node is already saved inside _setUserPermission()_